esp/services/ws_fs/ws_fsService.cpp

```diff
@@ -2298,6 +2264,66 @@ bool CFileSprayEx::onSprayVariable(IEspContext &context, IEspSprayVariable &req,
     return true;
 }
 
+void CFileSprayEx::checkDZScopeAccessAndSetSpraySourceDFUFileSpec(IEspContext &context, const char *srcPlane, const char *srcHost,
+    const char *srcFile, MemoryBuffer &srcXML, IDFUfileSpec *srcDFUfileSpec)
+{
+    if(srcXML.length() == 0)
+    {
+        RemoteMultiFilename rmfn;
+        SocketEndpoint ep(srcHost);
+        if (ep.isNull())
+            throw makeStringExceptionV(ECLWATCH_INVALID_INPUT, "Cannot resolve source network IP from %s.", srcHost);
+
+        rmfn.setEp(ep);
+        StringBuffer fnamebuf(srcFile);
+        fnamebuf.trim();
+        rmfn.append(fnamebuf.str());    // handles comma separated files
```

Comment:
question: what formats are supported as input for srcFile? Is it just a plain csv list of local files?
If so, RemoteMultiFilename, and getLocalPath etc. seems overkill vs just StringArray::appendList

Reply:
Will change